src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback } from 'react';
+import { EuiPage, EuiPageBody, EuiPageHeader, EuiPageContent, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WorkspaceForm, WorkspaceFormSubmitData } from './workspace_form';
+import { WORKSPACE_OVERVIEW_APP_ID, WORKSPACE_OP_TYPE_CREATE } from '../../../common/constants';
+import { formatUrlWithWorkspaceId } from '../../../../../core/public/utils';
+import { WorkspaceClient } from '../../workspace_client';
+
+export const WorkspaceCreator = () => {
+  const {
+    services: { application, notifications, http, workspaceClient },
+  } = useOpenSearchDashboards<{ workspaceClient: WorkspaceClient }>();
+
+  const isPermissionEnabled = application?.capabilities.workspaces.permissionEnabled;
+  const handleWorkspaceFormSubmit = useCallback(
+    async (data: WorkspaceFormSubmitData) => {
+      let result;
+      try {
+        const { permissions, ...attributes } = data;
+        result = await workspaceClient.create(attributes, permissions);
+      } catch (error) {
+        notifications?.toasts.addDanger({
+          title: i18n.translate('workspace.create.failed', {
+            defaultMessage: 'Failed to create workspace',
+          }),
+          text: error instanceof Error ? error.message : JSON.stringify(error),
+        });
+        return;
+      }
+      if (result?.success) {
+        notifications?.toasts.addSuccess({
+          title: i18n.translate('workspace.create.success', {
+            defaultMessage: 'Create workspace successfully',
+          }),
+        });
+        if (application && http) {
+          window.location.href = formatUrlWithWorkspaceId(
+            application.getUrlForApp(WORKSPACE_OVERVIEW_APP_ID, {
+              absolute: true,
+            }),
+            result.result.id,
+            http.basePath
+          );
+        }
+        return;
+      }
+      notifications?.toasts.addDanger({
+        title: i18n.translate('workspace.create.failed', {
+          defaultMessage: 'Failed to create workspace',
+        }),
+        text: result?.error,
+      });
+    },
+    [notifications?.toasts, http, application, workspaceClient]
+  );
+
+  return (
+    <EuiPage paddingSize="none">
+      <EuiPageBody>
+        <EuiPageHeader restrictWidth pageTitle="Create Workspace" />
+        <EuiSpacer />
+        <EuiPageContent
+          verticalPosition="center"
+          horizontalPosition="center"
+          paddingSize="none"
+          color="subdued"
+          hasShadow={false}
+          style={{ width: '100%', maxWidth: 1000 }}
```

Comment:
What is the `maxWidth: 1000` used for?

Reply:
This `maxWidth: 1000` is for align with the above page header title.
![image](https://github.com/ruanyl/OpenSearch-Dashboards/assets/4034161/29047a01-2741-4ed8-a92c-85f7a580c7c5)
It will strength full width if not set.

Reply:
I see, can we add a comment here? For someone without the context like me may be confused on this style property.